scripts/find_outliers.py

```diff
@@ -6,6 +6,10 @@
 """
 
 import sys
+import numpy as np
```

Comment:
The standard habit for imports is to put them in the following order:

* Python standard library imports such as `import os` `import sys`;
* other common imports like numpy, scipy or matplotlib;
* more specialized imports like nibabel;
* your own code imports.